src/main/java/weizberg/rijks/RijksFrame.java

```diff
@@ -0,0 +1,184 @@
+package weizberg.rijks;
+
+import com.andrewoid.ApiKey;
+import hu.akarnokd.rxjava3.swing.SwingSchedulers;
+import io.reactivex.rxjava3.disposables.Disposable;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import retrofit2.http.Url;
+import weizberg.rijks.json.ArtObject;
+import weizberg.rijks.json.ArtObjectsCollection;
+import weizberg.rijks.json.DisplayImageFrame;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import javax.swing.event.DocumentEvent;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class RijksFrame extends JFrame {
+    private JTextField searchField = new JTextField();
+    private ArtObjectsCollection artObjectsCollection;
+    private Button previousPageButton = new Button("Previous Page");
+    private Button nextPageButton = new Button("Next Page");
+    JPanel main = new JPanel();
+
+
+    JPanel imagePanel = new JPanel(new GridLayout(3, 4));
+
+    String[] titleAndArtist;
+    String[] imageLinks;
```

Comment:
You don't need to save these, save the array of `ArtObjects` or the `ArtObjectsCollection`